server/models.py

```diff
@@ -42,9 +42,22 @@ def __repr__(self):
         return f"<id {self.id}>"
 
     def json(self) -> Dict[str, str]:
+
+        if debug:

```

Comment:
You should be using the flask debug variable instead in order to avoid having to manage two different debug variables. It can be referenced via `current_app` instead.
```suggestion
        if current_app.debug:
```